tests/contenttypes_tests/test_management.py

```diff
@@ -22,6 +22,8 @@ class RemoveStaleContentTypesTests(TestCase):
 
     @classmethod
     def setUpTestData(cls):
+        with captured_stdout():
+            call_command('remove_stale_contenttypes', interactive=False, include_stale_apps=True, verbosity=2)
```

Comment:
@adamchainz  I'm currently pre-deleting all contenttypes before each contenttypes test to work around app labels being incorrectly selected as stale.

I have a feeling there's a better more elegant solution than this. One that will feel less astonishing and less like a hack.

P.S sorry for the double mention, accidentally pressed start a review instead of add single comment

Reply:
`setUpTestData` only runs once at the start of the test case, so I don't think this is particularly bad. But if we coudl avoid app labels being incorrectly marked as stale, tath would be better.

Reply:
If we keep this - instead of a fix for the stale app labels issue - it'd be worth a comment to explain why we're doing it.